cypress/integration/events_page.spec.ts

```diff
@@ -1,8 +1,24 @@
+import { ZetkinEvent, ZetkinEventResponse } from '../../src/types/zetkin';
+
 describe('/o/[orgId]/events', () => {
     beforeEach(() => {
         cy.request('delete', 'http://localhost:8001/_mocks');
     });
 
+    let dummyEventResponses : {data: ZetkinEventResponse[]};
+    let dummyEvents : {data: ZetkinEvent[]};
+
+    before(() => {
```

Comment:
I think it would be nice to put `before()` at the top instead of right here, so that the order of hooks is this:

* `before()`
* `beforeEach()` 
* `afterEach()` (not needed in this file, so can be ignored, but left here for the sake of argument)
* `after()`

That's the order in which they are invoked (the middle two for every test) and I think the tests would be more readable if that order was reflected in the order in which they appear in the file.